cron/cron.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+require_once __DIR__ . '/../class/webobserver.class.php';
+require_once __DIR__ . '/../class/jsonResponse.class.php';
+
+class WebObserverCron {
+
+	public $output;
+
+	function __construct()
+	{
+		global $langs, $conf;
+
+		if(empty($langs) || get_class($langs) !== 'Translate'){
+			$langs = new Translate('', $conf);
+		}
+
+		$langs->setDefaultLang(!empty($conf->global->MAIN_LANG_DEFAULT)?$conf->global->MAIN_LANG_DEFAULT:'en_US');
+		$langs->loadLangs(array('main', 'admin', 'cron', 'dict'));
+		if(is_callable(array($langs, 'reload'))){
+			$langs->reload('webobserver@webobserver');
+		}else{
+			$langs->load('webobserver@webobserver');
+		}
+	}
+
+	/**
+	 * Update instance data on linked wehbost
+	 * @return int|string Errors
+	 */
+	public function updateInstanceData()
+	{
+		global $conf, $langs;
+
+		$webObserver  = new WebObserver();
+
+		$webObserver->securityCheck();
+
+		$jsonResponse = new webObserver\JsonResponse();
+
+		$instanceData = $webObserver->getInstanceData();
+		$webHostUrl   = $conf->global->WEBOBSERVER_WEBHOST_URL;
+
+		if (dol_strlen($webHostUrl) > 0) {
```

Comment:
note pour moi même ajouter aussi un check d'url
```suggestion
		if (dol_strlen($webHostUrl) > 0 && filter_var($webHostUrl, FILTER_VALIDATE_URL)) {
```